WalletWasabi.Fluent/ViewModels/CoinControl/DataGridSource.cs

```diff
@@ -0,0 +1,147 @@
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Avalonia.Controls;
+using Avalonia.Controls.Models.TreeDataGrid;
+using Avalonia.Controls.Templates;
+using NBitcoin;
+using WalletWasabi.Fluent.Helpers;
+using WalletWasabi.Fluent.TreeDataGrid;
+using WalletWasabi.Fluent.ViewModels.CoinControl.Core;
+using WalletWasabi.Fluent.Views.CoinControl.Core.Cells;
+using WalletWasabi.Fluent.Views.CoinControl.Core.Headers;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinControl;
+
+public static class DataGridSource
```

Comment:
Can we rename it to CoinCointrolDataGridSource? Or something that indicates it is for coin control. DataGridSource is too generic.